.eslintrc

```diff
@@ -11,7 +11,7 @@
   "parserOptions": {
     "ecmaVersion": 2018, // Allows for the parsing of modern ECMAScript features
     "sourceType": "module", // Allows for the use of import
-    "project": "./tsconfig.eslint.json" // Allows for the use of rules which require parserServices to be generated
+    "project": "./tsconfig.json" // Allows for the use of rules which require parserServices to be generated
```

Comment:
@JoshuaKGoldberg What is the proper way to setup this in monorepos? Feels like I sometimes need the explicit `.eslint` file and sometimes it doesn't work at all 🤔

Reply:
Yeah it can get a bit tricky. I'd recommend [one tsconfig.json per package (and an optional one in the root)](https://typescript-eslint.io/linting/typed-linting/monorepos#one-tsconfigjson-per-package-and-an-optional-one-in-the-root).

The biggest sticking point is making sure _some_ TSConfig matches each linted file. That way typescript-eslint's can get TypeScript to give type information for that file.

LMK if playing around with that type of config doesn't work, and I can take a look!

Reply:
Something like this then? https://github.com/trpc/trpc/pull/3186/commits/ca79838d3dcfd0082c5ce8affc99ccdc5ac394b1

Would it also be okay with like `./examples/**/tsconfig.json` to take care of all the nested workspaces too?

Reply:
nvm I seem to have broken that parser setup. would you mind taking a look?

Reply:
Looks great to me! The only change is I added `./examples/*/*/tsconfig.json` instead so we don't get into the [wide globs issues](https://typescript-eslint.io/linting/typed-linting/monorepos#wide-globs-in-parseroptionsproject).

What parser issues are you seeing?